manualtesting/manual_testing.go

```diff
@@ -100,7 +100,9 @@ func manualTest(c *web.Context, w http.ResponseWriter, r *http.Request) {
 		}
 
 		c.App.Srv.Store.User().VerifyEmail(user.Id, user.Email)
-		<-c.App.Srv.Store.Team().SaveMember(&model.TeamMember{TeamId: teamID, UserId: user.Id}, *c.App.Config().TeamSettings.MaxUsersPerTeam)
+		if _, err := c.App.Srv.Store.Team().SaveMember(&model.TeamMember{TeamId: teamID, UserId: user.Id}, *c.App.Config().TeamSettings.MaxUsersPerTeam); err != nil {
```

Comment:
You are changing here the behavior, I think we should keep it as was before just removing the `<-`.

Reply:
So just ignore the output with `_`?

Reply:
yes and no, my idea is to ignore the output but just not assigning the output to anything. a line with just `c.App.Srv.Store.Team().SaveMember(&model.TeamMember{TeamId: teamID, UserId: user.Id}, *c.App.Config().TeamSettings.MaxUsersPerTeam)`

Reply:
alright